tethys_sdk/services.py

```diff
@@ -1 +1 @@
-"""
********************************************************************************
* Name: services.py
* Author: Nathan Swain
* Created On: 7 August 2015
* Copyright: (c) Brigham Young University 2015
* License: BSD 2-Clause
********************************************************************************
"""
# flake8: noqa
# DO NOT ERASE
from tethys_services.utilities import (
    list_dataset_engines,
    get_dataset_engine,
    list_spatial_dataset_engines,
    get_spatial_dataset_engine,
    list_wps_service_engines,
    get_wps_service_engine,
    ensure_oauth2
)
\ No newline at end of file
+"""
********************************************************************************
* Name: services.py
* Author: Nathan Swain
* Created On: 7 August 2015
* Copyright: (c) Brigham Young University 2015
* License: BSD 2-Clause
********************************************************************************
"""
# flake8: noqa
# DO NOT ERASE
from tethys_services.utilities import (
    list_dataset_engines,
    get_dataset_engine,
    list_spatial_dataset_engines,
    get_spatial_dataset_engine,
    list_wps_service_engines,
    get_wps_service_engine,
    ensure_oauth2,
)

```

Comment:
Not sure what is going on here.

Reply:
I'm not sure either. Is it possible this file was created on Windows and had `\n\r` characters?